masterlist.yaml

```diff
@@ -5243,6 +5248,89 @@ plugins:
+        subs: [ 'Even Better Quest Objectives' ]
+        condition: 'active("BetterQuestObjectives.esp") and not active("EconomyOverhaul CV - EvenBetterQuestObjectives Patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Missives' ]
+        condition: 'checksum("Missives.esp",12647D6D) and not active("EconomyOverhaul CV - Missives Patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Missives Open Cities' ]
+        condition: 'checksum("Missives.esp",2C169E8B) and not active("EconomyOverhaul CV - MissivesOpenCities Patch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Missives 2.0 or later' ]
+        condition: 'version("Missives.esp", "2.0", >=) and not active("EconomyOverhaul CV - Missives Patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Morrowloot Ultimate' ]
+        condition: 'active("MLU.esp") and not active("EconomyOverhaul CV - MorrowLootUltimate Patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Ordinator - Perks of Skyrim' ]
+        condition: 'active("Ordinator - Perks of Skyrim.esp") and not active("EconomyOverhaul CV - Ordinator Patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Quests Are In Skyrim' ]
+        condition: 'active("QuestsAreInSkyrimUSSEP.esp") and not active("EconomyOverhaul CV - QuestsAreinSkyrim Patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Scarcity' ]
+        condition: 'active("Scarcity SE - Less Loot Mod.esp") and not active("EconomyOverhaul CV - Scarcity Patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Trade and Barter' ]
+        condition: 'active("Trade & Barter.esp") and not active("EconomyOverhaul CV - TradeandBarter Patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Weapons Armor Clothing and Clutter Fixes' ]
+        condition: 'active("WACCF_Armor and Clothing Extension.esp") and not active("EconomyOverhaul CV - WACCF Patch.esp")'
+    clean:
+      # version: 2.1
+      - crc: 0x146D569B
+        util: 'SSEEdit v4.0.2'
+  - name: 'EconomyOverhaulandSpeechcraftImprovementsLite.esp'
+    msg:
+      - <<: *patchIncluded
+        subs: [ 'Cutting Room Floor' ]
+        condition: 'active("Cutting Room Floor.esp") and not active("EconomyOverhaul LV - CuttingRoomFloor Patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Even Better Quest Objectives' ]
+        condition: 'active("BetterQuestObjectives.esp") and not active("EconomyOverhaul LV - EvenBetterQuestObjectives Patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Missives' ]
+        condition: 'checksum("Missives.esp",12647D6D) and not active("EconomyOverhaul LV - Missives Patch.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Missives Open Cities' ]
+        condition: 'checksum("Missives.esp",2C169E8B) and not active("EconomyOverhaul LV - MissivesOpenCities Patch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Missives 2.0 or later' ]
+        condition: 'version("Missives.esp", "2.0", >=) and not active("EconomyOverhaul LV - Missives Patch.esp")'
```

Comment:
Is the version included in the Missive plugin?

Reply:
I thought the version showing up in Mod Organizer 2 was parsed from the header. Since I am going to delete the other messages, this version check condition will be no longer needed anyway. I will update it.